engine/src/flutter/lib/ui/window/platform_configuration.cc

```diff
@@ -552,6 +571,17 @@ Dart_Handle PlatformConfigurationNativeApi::SendPortPlatformMessage(
   return HandlePlatformMessage(dart_state, name, data_handle, response);
 }
 
+void PlatformConfigurationNativeApi::RequestViewFocusChange(int64_t view_id,
+                                                            int64_t state,
+                                                            int64_t direction) {
+  ViewFocusChangeRequest request{view_id,  //
```

Comment:
Missing comment or should the `//` be removed?

Reply:
The comment is there to force line break. Otherwise clang format puts first two argument in same line which doesn't look very readable. It's used in other parts of the engine as well.